src/components/cursor-controller.js

```diff
@@ -96,7 +156,10 @@ AFRAME.registerComponent("cursor-controller", {
       }
 
       if (this.data.drawLine) {
-        this.el.setAttribute("line", { start: this.origin.clone(), end: this.data.cursor.object3D.position.clone() });
+        this.el.setAttribute("line", {
```

Comment:
For all the reasons you stated re updated the raycaster aframe properties per frame we should just be directly using a `THREE.Line` here as well, but would be fine as its own PR.

Reply:
ditto https://github.com/mozilla/hubs/issues/421

Reply:
Yes, that will improve performance in the case where the line is active. I will do it in a future PR.